homeassistant/components/v2c/switch.py

```diff
@@ -80,11 +79,11 @@ class V2CSwitchEntityDescription(SwitchEntityDescription):
 
 async def async_setup_entry(
     hass: HomeAssistant,
-    config_entry: ConfigEntry,
+    config_entry: V2CConfigEntry,
     async_add_entities: AddEntitiesCallback,
 ) -> None:
     """Set up V2C switch platform."""
-    coordinator: V2CUpdateCoordinator = hass.data[DOMAIN][config_entry.entry_id]
+    coordinator: V2CUpdateCoordinator = config_entry.runtime_data
```

Comment:
You no longer need the type because runtime_data is typed.
```suggestion
    coordinator = config_entry.runtime_data
```